packages/manager/apps/web-domains/public/translations/allDom/Messages_fr_FR.json

```diff
@@ -49,5 +50,9 @@
   "allDom_status_automatic": "Automatique",
   "allDom_status_manual": "Manuel",
   "allDom_automatic_renew": "Renouvellement prévu le",
-  "allDom_manual_renew": "Date d'expiration"
+  "allDom_manual_renew": "Date d'expiration",
+  "allDom_cancel_terminate_title": "Annuler la résiliation du service {{t0}}",
```

Comment:
The var inside the trad key can be named, I think this is easier to read in the instantiation of the key